packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices.js

```diff
@@ -0,0 +1,79 @@
+import {
+  Grid
+} from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import sortIntoColumns from 'sort-into-columns'
+import styled, { css } from 'styled-components'
+
+import ChoiceButton from './ChoiceButton'
+
+const StyledGrid = styled(Grid)`
+  ${props => props.theme.dark ?
+    css`background-color: ${props.theme.global.colors['dark-1']};` :
+    css`background-color: ${props.theme.global.colors['light-1']};`
+  }
+`
+
+function howManyColumns ({ length }) {
+  if (length <= 5) {
+    return 1
+  } else if (length <= 20) {
+    return 2
+  } else {
+    return 3
+  }
+}
+
+function Choices (props) {
+  const {
+    filteredChoices,
+    onChoose,
+    task
+  } = props
+
+  const columnsCount = howManyColumns(filteredChoices)
+  const sortedFilteredChoices = sortIntoColumns(filteredChoices, columnsCount)
+
+  return (
+    <StyledGrid
+      columns={{
+        count: columnsCount,
+        size: 'auto'
+      }}
+      gap='2px'
+    >
```

Comment:
PFE uses [`flex-basis` applied to all children](https://github.com/zooniverse/Panoptes-Front-End/blob/master/css/survey-task.styl#L119-L127) of the containing element, depending on a column property. I originally implemented the same as PFE, and can easily revert accordingly, but on a second try using `<Grid />` it worked very cleanly, so running with `<Grid />` initially, but very open to suggestion.

Reply:
Grids support [auto-placement by column](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout#auto-placement_by_column), which would allow us to get rid of the old hack for splitting the choices list into smaller column lists. My initial vote would be for grids.